src/App.tsx

```diff
@@ -15,36 +22,80 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+function getPrepearedGood(
```

Comment:
Typo: The function name `getPrepearedGood` should be corrected to `getPreparedGoods` for clarity and consistency.